libs/langchain/langchain/llms/google_palm.py

```diff
@@ -117,31 +173,164 @@ def _generate(
         prompts: List[str],
         stop: Optional[List[str]] = None,
         run_manager: Optional[CallbackManagerForLLMRun] = None,
+        stream: Optional[bool] = None,
```

Comment:
We should just put the `stream` method for this now that it's available

Reply:
I'm not sure I fully got your point. I've added the `_stream` method, haven't I?

Reply:
Yeah that's correct. What we don't need (or want) to support is streaming via the `_generate` method, since that's meant to be a single invocation. 
Similarly, `streaming` shouldn't be initialized on the LLM itself -> it should only be supported via the stream() method

Reply:
thanks for the explanation, fixed.